packages/react-query-5/src/SuspenseQuery.tsx

```diff
@@ -7,6 +7,21 @@ import {
 } from '@tanstack/react-query'
 import type { ReactNode } from 'react'
 
+/**
+ * We provide these components to clearly express what causes suspense at the same depth.
+ * `<SuspenseQuery/>` serves to make `useSuspenseQuery` easier to use in jsx.
+ * @see {@link https://suspensive.org/en/docs/react-query/SuspenseQuery Suspensive Docs}
+ * @example
+ * ```tsx
+ * import { SuspenseQuery } from '@suspensive/react-query'
+ *
+ * // You can use QueryOptions as props.
+ * <SuspenseQuery {...queryOptions()}>
+ *   {({ data, isLoading }) => { // QueryResult can be used in children.
+ *     return <></>
+ *   }
+ * </SuspenseQuery/>
```

Comment:
The examples are designed to be straightforward, allowing users to grasp the usage without significant complexity (ensuring minimal impact in case of future changes).

This is just a suggestion. If it seems that the example won’t align with my intention, I can consider deleting it as well.

Reply:
It's good enough! Let's try it and improve it if need!